plugins/osx/osx.plugin.zsh

```diff
@@ -161,6 +161,15 @@ function itunes() {
 		vol)
 			opt="set sound volume to $1" #$1 Due to the shift
 			;;
+		playing)
+			local state=`osascript -e 'tell application "iTunes" to player state as string'`
+			if [ "$state" = "playing" ]; then
+				currenttrack=`osascript -e 'tell application "iTunes" to name of current track as string'`
+				currentartist=`osascript -e 'tell application "iTunes" to artist of current track as string'`
+				echo "You're currently litening to $fg[yellow]$currenttrack$reset_color by $fg[yellow]$currentartist$reset_color";
```

Comment:
Confirmed: works for me on OS X 10.9.5.

<img width="853" alt="screen shot 2015-09-16 at 3 28 24 pm" src="https://cloud.githubusercontent.com/assets/2618447/9915893/b4bfb79a-5c87-11e5-8491-7f4ecd83f37c.png">

If you use the prompt expansion colors, you will need to escape the song and artist names in case they contain "%" or other prompt expansion escape characters. As it is, with `echo`, you should probably use `echo -E` in case they contain backslashes, too.

And if you want to be super hygienic, check to make sure the output is connected to a tty using `[ -t 1 ]` and only use the color control sequences if you're in fact outputting to a tty, to avoid contaminating output captured to files or other variables.